public/app/plugins/panel/geomap/layers/basemaps/default.ts

```diff
@@ -0,0 +1,38 @@
+import { MapLayerRegistryItem, MapLayerOptions, GrafanaTheme2 } from '@grafana/data';
+import Map from 'ol/Map';
+import { carto } from './carto';
+import { esriXYZTiles } from './esri';
+import { xyzTiles } from './generic';
+import { standard } from './osm';
+import { config } from 'app/core/config';
+
+// Array of base map options to search through
+const baseLayers = [carto, esriXYZTiles, xyzTiles, standard];
+
+// Configured base map from the server side
+const defaultBaseLayer = config.geomapDefaultBaseLayer;
```

Comment:
just use `config.geomapDefaultBaseLayer;`  rather than create a new variable for it